Content.Server/GameTicking/GameTicker.cs

```diff
@@ -774,6 +774,10 @@ private async void SpawnPlayer(IPlayerSession session, string jobId = null, bool
             AddManifestEntry(character.Name, jobId);
             AddSpawnedPosition(jobId);
             EquipIdCard(mob, character.Name, jobPrototype);
+            if(jobPrototype.ID == "Clown")
```

Comment:
Yeah this isn't gonna fly.

Best solution here is to allow custom C# logic to be specified from the job prototype and then doing this there.

Reply:
ah, nvm then.